docs/gulp/tasks/jshint.js

```diff
@@ -0,0 +1,7 @@
+var gulp = require('gulp');
+var shell = require('gulp-shell');
+var handleErrors = require('../util/handleErrors');
+
+gulp.task('jshint', shell.task([
+  '../node_modules/.bin/jsxhint --harmony ../src/* ../src/**/* ../src/**/**/* ./src/app/* ./src/app/**/* ./src/app/**/**/* ./src/app/**/**/**/* --exclude ../src/utils/modernizr.custom.js'
```

Comment:
Try:
```javascript
'../node_modules/.bin/jsxhint --harmony \'../src/**\' --exclude ../src/utils/modernizr.custom.js'
```
Note the escaped single quotes around the glob. jsxhint uses glob-all: https://github.com/jpillora/node-glob-all